packages/react-core/src/components/Badge/examples/Badge.md

```diff
@@ -16,3 +16,8 @@ propComponents: ['Badge']
 
 ```ts file="./BadgeUnread.tsx"
 ```
+
+### Badge as toggle
+
+```ts file="./BadgeToggle.tsx"
+```
```

Comment:
@edonehoo For this example, we want to convey that a badge with toggle *must* be passed to a MenuToggle component (and maybe link to anoher component's example to show how that's done?). What do you think of the following:

"You can use a badge as a toggle by passing the `isToggle` property. You **must** then pass the badge as a child to our menu toggle component" (could also link tot he menu toggle page here)

Reply:
I think this sounds perfect! Would also link to a menu toggle example like you suggest. this one? > /components/menus/menu-toggle#with-a-badge

Reply:
I guess we don't have a MenuToggle example that uses a badge as a toggle itself... We have this [breadcrumb with dropdown](https://www.patternfly.org/components/breadcrumb#with-dropdown) example, though; maybe worth adding a demo or something somewhere in either MenuToggle or Badge to show how to use a toggle variant Badge.

For now I'll open a followup for the example description, though, just to prevent holding this PR up.